runtime/oti/VMHelpers.hpp

```diff
@@ -2058,6 +2058,16 @@ class VM_VMHelpers
 		return oldState;
 	}
 
-};
+#if JAVA_SPEC_VERSION >= 20
+	static VMINLINE void
+	virtualThreadHideFrames(J9VMThread *currentThread, j9object_t vthread, bool hide) {
```

Comment:
since vthread object is not used, we don't need to pass it. Also if you use `jboolean` instead of `bool`, then you can directly pass the `hide` argument rather than explicit `true/false`.

Reply:
Does the order of calling hideframes and mountBegin/End matter? Can I move it outside of the `if` statement?